src/tests/ftest/pool/evict_test.py

```diff
@@ -38,110 +39,257 @@ class EvictTest(TestWithServers):
+            if pool_info:
+                if c_uuid_to_str(pool_info.pi_uuid) == pool[count].uuid:
+                    self.log.info(
+                        "Pool # %s UUID matches pool_info.pi_uuid %s",
+                        count+1, pool[count].uuid)
+                else:
+                    self.fail(
+                        "Pool # {} UUID does not matches pool_info.pi_uuid:  "
+                        "{} != {}".format(
+                            count+1, pool[count].uuid, c_uuid_to_str(
+                                pool_info.pi_uuid)))
+
+    def test_evict_bad_uuid(self):
         """
-        Test connecting to a pool.
+        Test evicting a pool with an invalid uuid.
 
         :avocado: tags=pool,poolevict
         """
-        # test parameters are in the EvictTest.yaml
-        setid = self.params.get("setname", '/run/tests/setnames/validsetname/*')
-        mode = self.params.get("mode", '/run/tests/modes/modeall/*')
-        uid = os.geteuid()
-        gid = os.getegid()
+        # Create a pool
+        self.log.info("Create a pool")
+        self.pool = TestPool(self.context, self.log)
+        self.pool.get_params(self)
+        self.pool.create()
+        self.log.info("Valid Pool UUID is %s", self.pool.uuid)
 
-        try:
-            create_cmd = (
-                "{0} create-pool "
-                "-m {1} "
-                "-u {2} "
-                "-g {3} "
-                "-s {4} "
-                "-c 1".format(self.daosctl, mode, uid, gid, setid))
-            uuid_str = """{0}""".format(process.system_output(create_cmd))
-            print("uuid is {0}\n".format(uuid_str))
-
-        except Exception as excep:
-            print(excep)
-            print(traceback.format_exc())
-            self.fail("Unexpected pool create and connect failure.\n")
-        try:
-            # use the wrong uuid, which of course should fail
-            bogus_uuid = "44be695840f4b6eda581b461a4d4c570490ba4db"
-            bad_evict_cmd = (
-                "{0} evict-pool "
-                "-i {1} "
-                "-s {2}".format(self.daosctl, bogus_uuid, setid))
-            process.system(bad_evict_cmd)
-
-        except Exception as excep:
-            # this section of the test should fail and throw an exception
-            print("command expected to fail")
+        # Check that the pool was created
+        self.assertTrue(
+            self.pool.check_files(self.hostlist_servers),
+            "Pool data not detected on servers before evict")
+
+        # Connect to the pool
+        self.assertTrue(self.pool.connect(1),
+                        "Pool connect failed before evict")
+        self.log.info(
+            "Pool UUID is %s and handle is %s",
+            self.pool.uuid, self.pool.pool.handle.value)
 
+        # Attempt to evict pool with invald UUID
+        saved_uuid = self.pool.uuid
+        bogus_uuid = '81ef94d7-a59d-4a5e-935b-abfbd12f2105'
```

Comment:
May want to use uuid.uuid4() or all f's in case this string happens to be the actual pool uuid.